src/menus/TrackerMenu/TrackerMenu.js

```diff
@@ -93,6 +80,30 @@ class TrackerMenu extends Component {
     });
   }
 
+  initializeClick() {
+    const { layerService } = this.props;
+    this.trackerLayers = layerService
+      .getLayersAsFlatArray()
+      .filter((l) => l.isTrackerLayer);
+
+    this.trackerLayers.forEach((layer) => {
+      layer.unClick(this.onLayerClick);
+    });
+
+    unByKey(this.olEventsKeys);
+    this.olEventsKeys = [];
+    this.trackerLayers.forEach((layer) => {
+      this.olEventsKeys.push(
+        layer.olLayer.on('change:visible', () => {
+          this.setState({
+            open: false,
+          });
+        }),
+      );
+      layer.onClick(this.onLayerClick);
```

Comment:
What about adding 
`layer.unClick(this.onLayerClick);`
here?

Then we only need one forEach.